### ISSUE TITLE
TrainAndEvaluateModel instance throws error when the method train_and_evaluate is called [tutorial_tf2.ipynb]

### ISSUE THREAD

**Issue by Senejohnny** [sebp/survival-cnn-estimator#2]
Hi Sebastian,

I get the below error when I run the notebook tutorial_tf2.ipynb. Could you please help me with this error?
![image](https://user-images.githubusercontent.com/46810505/129054961-0cebb3ad-faec-4b19-80de-bafd374be4e7.png)

**Comment by Senejohnny** [comment]
Apparently the [graph method is updated](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/summary_ops_v2.py) so changing graph to graph_v1 should solve the error. 
This solved the error for me

**Comment by sebp** [comment]
Yes, they changed `graph` to `graph_v1` in [this commit](https://github.com/tensorflow/tensorflow/commit/fbe14f3bf379e33d14f35f914db56143e817edda#diff-d2cf8455ca3dbffe505170fa5fe38ed5e0d5969bcacb3b92e0b638b6d3c20461), which is included in TF2.5 and later.
